next.config.js

```diff
@@ -3,10 +3,20 @@ const { parsed: localEnv } = require("dotenv").config();
 const webpack = require("webpack");
 const apiKey = JSON.stringify(process.env.SHOPIFY_API_KEY);
 const appUrl = process.env.APP_URL;
+const hgApiUrl = process.env.HG_API_URL;
+const hgApiKey = process.env.HG_API_KEY;
+const hgEnvId = process.env.HG_ENV_ID;
+const hgFlowId = process.env.HG_FLOW_ID;
+const hgFlowVersion = process.env.HG_FLOW_VERSION ?? "1";
 
 module.exports = {
   env: {
     APP_URL: appUrl,
+    HG_API_URL: hgApiUrl,
+    HG_API_KEY: hgApiKey,
```

Comment:
Davi doesn't this mean the secret API key would be exposed in the frontend? This would be bad right?

Reply:
Yes 😢. I think it's bad indeed.

But this was solution I thought it was going to make it work... Does the public key works for fetching linked sources with config?

Reply:
No it does not. Why do we need to get the linked source config? Can't we just get any tenant with the prefix and assume that tenant is the correct one?

Reply:
Well, yes, I think we can... At least this is going to be the ideal. I'll implement that shortly